src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -143,16 +143,13 @@ class Simpletask : ThemedActivity() {
         }
         localBroadcastManager!!.registerReceiver(m_broadcastReceiver, intentFilter)
 
-
-        // Set the proper theme
-        setTheme(Config.activeTheme)
+        // Landscape mode
         if (Config.hasLandscapeDrawers()) {
             setContentView(R.layout.main_landscape)
             val toolbar = findViewById(R.id.saved_filter_toolbar) as Toolbar
             toolbar.setOnMenuItemClickListener{ item ->
                 onOptionsItemSelected(item)
             }
-            toolbar.popupTheme = Config.activeTheme
```

Comment:
If we don't set this, toolbar.popupTheme is inherited from the toolbar's main theme. With the new themes, we no longer need special styling on menus, so this can go.